.github/workflows/weekly.yml

```diff
@@ -0,0 +1,73 @@
+name: build branch
+
+on:
+  schedule:
+    - cron: '0 0 * * 0' # weekly
+
+jobs:
+
+  test:
+    runs-on: ubuntu-latest
+    name: Check branch
+    steps:
+    - uses: actions/checkout@v2
+
+    # Install Node 12
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 12
+
+    # Cache node modules
+    - uses: actions/cache@v2
+      env:
+        cache-name: cache-node-modules
+      with:
+        # npm cache files are stored in `~/.npm` on Linux/macOS
+        path: ~/.npm
+        key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-build-${{ env.cache-name }}-
+          ${{ runner.os }}-build-
+          ${{ runner.os }}-
+
+    # Install environment
+    - run: npm config set spin false
+    - run: npm install -g npm@7
+    - run: npm install -g codecov
+    - run: npm --version
+
+    # Install Dependencies
+    - run: npm ci
+
+    # Run tests
+    - run: npm run lint:js
+    - run: npm run lint:hbs
+    - run: npm test
+    - run: codecov --file=coverage/node/lcov.info
```

Comment:
I think we can use this action here:

https://github.com/codecov/codecov-action